packages/components/src/components/navigation/SubNav.tsx

```diff
@@ -152,12 +147,7 @@ export class SubNav extends React.Component<ISubNavProps, SubNavState> {
                     <div className="tab-scroll-ct" ref={this.scrollable}>
                         <ul className="nav navbar-nav">
                             {tabs.map((tab, i) => (
-                                <NavItem
-                                    key={i}
-                                    to={tab.url}
-                                    onActive={this.state.isScrollable && this.onItemActivate}
-                                    onlyActiveOnIndex={tab.text === 'Overview'}
-                                >
+                                <NavItem key={i} to={tab.url} onActive={this.onItemActivate}>
```

Comment:
I don't know off the top of my head why the previous code had the `this.state.isScrollable` check, but just curious why that was removed?

Reply:
It was a somewhat incorrect over-optimization. The `onItemActivate` method is already checking for `this.state.isScrollable` so I just let the `onActive` be set to the method all the time. The incorrectness came from `onActive={this.state.isScrollable && this.onItemActivate}` would equal `false` when `!this.state.isScrollable` which is not a function.